packages/rpc-spec/src/rpc.ts

```diff
@@ -72,6 +72,9 @@ function makeProxy<TRpcMethods, TRpcTransport extends RpcTransport>(
             return false;
         },
         get(target, p, receiver) {
+            if (p === 'then') {
```

Comment:
Note for myself: this doesn't have to be `p.toString()` because there's no way of calling the `then` method of a `Promise` using a `Symbol`.